Makefile

```diff
@@ -35,15 +35,15 @@ prepare: gomod
 
 gomod:
 	$(GO) env -w GO111MODULE=on
-	$(GO) env -w GOPROXY=https://goproxy.io,direct
-	$(GO) env -w CGO_ENABLED=0
+	$(GO) env -w GOPROXY=https://goproxy.cn,direct
+	$(GO) env -w CGO_ENABLED=1
```

Comment:
CGO_ENABLED=1 causes dynamic link, and cannot run on Alpine OS.

Reply:
update done